src/open_inwoner/cms/cases/views/status.py

```diff
@@ -221,6 +224,19 @@ def get_initiator_display(self, case: Zaak) -> str:
             return ""
         return ", ".join([get_role_name_display(r) for r in roles])
 
+    def get_statuses_data(
+        self, statuses: List[Status], lookup: TranslationLookup
+    ) -> List[dict]:
```

Comment:
You can use the built-in type `list` for the annotation since we don't need to support Python < 3.9